pages/linux/raspi-config.md

```diff
@@ -0,0 +1,8 @@
+# raspi-config
+
+> An ncurses terminal GUI to config a Raspberry Pi.
+> More information: <https://www.raspberrypi.org/documentation/computers/configuration.html>.
+
+- Start raspi-config:
```

Comment:
```suggestion
- Start `raspi-config`:
```

Reply:
Not to happy with this one, since its not really a standard or frequent used command and many other pages don't use backticks around things like this.

Reply:
@CleanMachine1 should we apply this?